Bottom: d427a0e3cbaccbb45fedf91d82fbed8d677eaadd
Top:    6407b661caae54ee405e3240a5b1ab5359578190
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 20:12:24 +0000

Refresh of tmp-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 4351a78..f0b70ef 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1703,9 +1703,9 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
   VNODE *cur;
   DIRECTION dir = UNINITIALISED;
 
-//  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
+  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
 //  cur = (&a->head); //    ->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
-  cur = (&a->head)        ;
+//  cur = (&a->head)        ;
   do
     {
 #if 0
@@ -1818,6 +1818,7 @@ M_B_AREA_Collect (jmp_buf * e, POLYAREA * bfst, POLYAREA ** contours,
 	       * NB: There Could be grief if the JUMP rule is inconsistent in
 	       *     its interpretation from each side of the vertex.
 	       */
+#if 1
             switch (action)
               {
               case PBO_UNITE:
@@ -1833,6 +1834,7 @@ M_B_AREA_Collect (jmp_buf * e, POLYAREA * bfst, POLYAREA ** contours,
                 Collect ('B', e, *cur, contours, holes, SubS_Rule, SubJ_Rule);
                 break;
               }
+#endif
 	    }
           else if ((*cur)->Flags.status == INSIDE)
 	    switch (action)
